CITATION.bib

```diff
@@ -0,0 +1,11 @@
+@article{Julia.jl,
+    title={Julia: A fresh approach to numerical computing},
+    author={Bezanson, Jeff and Edelman, Alan and Karpinski, Stefan and Shah, Viral B},
+    journal={SIAM {R}eview},
+    volume={59},
+    number={1},
+    pages={65--98},
+    year={2017},
+    publisher={SIAM},
+    url={https://doi.org/10.1137/141000671}
```

Comment:
Please, add the field `doi = 10.1137/141000671`, it's correctly used by many biblatex styles